app/services/language-preference.js

```diff
@@ -1,10 +1,8 @@
 import { run } from '@ember/runloop';
-import { getOwner } from '@ember/application';
 import { hash } from 'rsvp';
 import Service, { inject as service } from '@ember/service';
 import Ember from 'ember';
 import config from '../config/environment';
-import { walkConfigs, setRTL } from '../utils/locale-utils';
```

Comment:
These were removed along with the corresponding files since they don't relate to `ember-intl` anymore and were caught in my find/replace for `ember-i18n` to `ember-intl`. Is there a way I can verify that RTL is working properly with `ember-intl` in the same way it was before with `ember-i18n`?

Reply:
I think there was one included locale that used that, Urdu. I remember it caused the entire nav bar to move to the right side of the screen instead of left. There's this https://github.com/HospitalRun/hospitalrun-frontend/issues/1247.